build.env

```diff
@@ -41,7 +41,8 @@ VM_DRIVER=none
 CHANGE_MINIKUBE_NONE_USER=true
 
 # Rook options
-ROOK_VERSION=v1.3.9
+ROOK_VERSION=v1.4.9
+ROOK_CEPH_CLUSTER_VERSION=v14.2.12
```

Comment:
hmm, this looks a little like a duplicate... `BASE_IMAGE` contains the version after the `:`, in case `ROOK_CEPH_CLUSTER_VERSION` is not set, maybe we can use the version from there?

Reply:
@agarwal-mudit cant we stick to the version of Ceph Cluster which ROOK installs with a specified version ?  Isnt it good to make sure we are sticking to the stability of compatibility from Rook?

Reply:
@humblec sometimes Rook does not use the ceph version which is required for cephcsi. It's good to stick to the version that we want from the cephcsi side. and this avoids the dependency on the Rook. we always install the ceph version that we want.

Reply:
Yeah, particularly in this case where rook is installing a ceph version which doesn't support retain snapshot feature

Reply:
@Madhu-1 @agarwal-mudit sure.. I was thinking that, if at all specific version is required in Developer system we could upgrade ceph with minimal hack in the backend instead of making the choice default. But its fine to have a tunable option too :+1: .. Thanks!